FordLabs/peoplemover
# Consolidate product card tests

## Issue
Resolves N/A

## What was done
- [x] Moved all tests out of Product.test.tsx that tested the ProductCard into the ProductCard.test.tsx file
- [x] Ensure all tests in Product.test.tsx that rendered the whole app were tested in cypress or other unit tests and deleted the file
- [x] Added an archived product to the default Flipping Sweet test space so that we could test archived products easier in cypress 

## How to test
1. Ensure that all tests that were in Product.test.tsx that tested the ProductCard are now in ProductCard.test.tsx
2. Ensure that all things tested in Product.test.tsx that required rendering the entire app are now only tested in cypress.
